app/assets/javascripts/components/shared/modal.js

```diff
@@ -24,6 +24,11 @@
         showTitle: false,
         // Title of the modal – mandatory for accessibility
         title: 'Untitled modal',
+        // Allows the modal container have scroll
+        allowScroll: false,
```

Comment:
We are: this is the way to allow scroll the page despite of having the modal opened. But this doesn't happen in the rest of modals, you can not scroll once the modal is opened.